.golangci.yml

```diff
@@ -24,16 +24,16 @@ linters:
     - govet
     - ineffassign
     # - interfacer
-    - lll
+    # - lll
     # - maligned
-    # - misspell
+    - misspell
```

Comment:
I kind of think that we shouldn't change the linter config on the branch.

Reply:
I'm not changing it, I'm just sync'ing with `master`. See https://github.com/tendermint/tendermint/blob/d433ebe68de20457d6cc0fd77a67ac5a7dbf5216/.golangci.yml#L29